gpio_sim/kernel_build_debian.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# For Debian 
+# Build and install GPIO-SIM kernel module
+# Main reference https://www.debian.org/doc/manuals/debian-kernel-handbook/ch-common-tasks.html
+
+# Install prerequisites
+sudo apt-get update
+sudo apt-get -y install dpkg-dev
+sudo apt-get -y install build-essential fakeroot
+sudo apt-get -y build-dep linux
+
+# Get linux source
+sudo apt-get -y install linux-source
```

Comment:
This is meant to be run on development machines to simulate the C3 GPIO. Unfortunately we can't have matching kernel version to the real C3 because the GPIO-SIM module requires at least version 5.19. So I had it grab the latest kernel instead.